src/relations/peers.py

```diff
@@ -289,6 +289,13 @@ def update_auth_file(self, auth_file: str) -> None:
         self.set_secret("app", AUTH_FILE_DATABAG_KEY, auth_file)
         logger.debug("updated auth file in peer databag")
 
+    def get_auth_file(self) -> str:
```

Comment:
Is this code being used in any part of the other files? I couldn't find its usage.